client/app/lib/kodingappscontroller.coffee

```diff
@@ -29,7 +53,9 @@ module.exports = class KodingAppsController extends KDController
     app = globals.config.apps[name]
 
     @putAppScript app, (err, res) =>
-      AppClass = require res.app.identifier
```

Comment:
webpack works by overriding `require` methods. It thinks that this is a regular commonjs `require`, but in fact it's a AMD module `require` which has the capability of working dynamically. We were using this statement to lazy load/async load app modules via `require.js`, but it's not that straight forward with `webpack`. It's doable tho, but i didn't want to spend anymore time on it just yet.